src/option.ts

```diff
@@ -1,10 +1,21 @@
 import { Options } from 'markdown-it'
-import { TextDocument, workspace } from 'vscode'
+import { coerce, lt } from 'semver'
+import { TextDocument, version, workspace } from 'vscode'
 import { MarpOptions } from '@marp-team/marp-core'
 
 let cachedPreviewOption: MarpOptions | undefined
 let cachedCLIOption: any
 
+// WebKit polyfill requires in VS Code < 1.36 (Electron 3).
+//
+// NOTE: Electron 3 has got a stable rendering by applying WebKit polyfill. And
+// Electron 4 has almost stable rendering even if polyfill is not used but still
+// remains glitch when used CSS 3D transform and video component. Electron 5
+// also has a glitch in video, and we have to wait for stable rendering until
+// Electron 6.
+const coercedVer = coerce(version)
+const isRequiredPolyfill = coercedVer ? lt(coercedVer, '1.36.0') : false
```

Comment:
VS Code seems not to provide API for getting Electron version. Instead we are using known VS Code version.